server/src/utils/account.js

```diff
@@ -10,8 +10,15 @@ const lockAccount = async (query = { role: '*' }) => {
   if (account) {
     console.log(`account ${account.address} with id ${account._id} is locked by query ${JSON.stringify(query)}`)
     return account
+  } else {
+    console.log(`no free account found by query ${JSON.stringify(query)}`)
+    // If account is locked more than maxLockMinutes, release account
+    var time = new Date();
+    time.setSeconds(time.getSeconds() - config.get('accounts.maxLockSeconds'));
```

Comment:
you can 
const time = new Date().time.setSeconds(time.getSeconds() - config.get('accounts.maxLockSeconds'));